### PR TITLE
Nh swerve abs

### PR DESCRIPTION
**Pull request by noahheller**

### CODE REVIEW

File: src/main/java/frc/robot/swervev2/encoder/SwerveAnalogAbsEncoder.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package frc.robot.swervev2.encoder;
+
+import edu.wpi.first.wpilibj.AnalogEncoder;
+import frc.robot.utils.math.AngleUtils;
+
+public class SwerveAnalogAbsEncoder implements SwerveAbsEncoder {
+
+    private final AnalogEncoder absEncoder;
+    private double steerOffset;
```
Comment (armadilloz) [inline src/main/java/frc/robot/swervev2/encoder/SwerveAnalogAbsEncoder.java]: set to 0.0

File: src/main/java/frc/robot/swervev2/encoder/SwerveCANAbsEncoder.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package frc.robot.swervev2.encoder;
+
+import com.ctre.phoenix.sensors.CANCoderStatusFrame;
+import com.ctre.phoenix.sensors.WPI_CANCoder;
+import frc.robot.utils.math.AngleUtils;
+
+public class SwerveCANAbsEncoder implements SwerveAbsEncoder {
+
+    private final WPI_CANCoder absEncoder;
+    private double steerOffset;
```
Comment (armadilloz) [inline src/main/java/frc/robot/swervev2/encoder/SwerveCANAbsEncoder.java]: set to 0.0

**Review by armadilloz — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/swervev2/encoder/SwerveAnalogAbsEncoder.java (hunk 1 above)
Comment (noahheller) [reply src/main/java/frc/robot/swervev2/encoder/SwerveAnalogAbsEncoder.java]: done

File: src/main/java/frc/robot/swervev2/encoder/SwerveCANAbsEncoder.java (hunk 2 above)
Comment (noahheller) [reply src/main/java/frc/robot/swervev2/encoder/SwerveCANAbsEncoder.java]: done

**Review by armadilloz — APPROVED** [review]

**Pull request merged**